extensions/wikia/WikiFactory/Dumps/runBackups.php

```diff
@@ -159,9 +160,12 @@ function doDumpBackup( $row, $path, array $args = [] ) {
 	$time = Wikia::timeDuration( wfTime() - $time );
 
 	Wikia::log( __METHOD__, "info", "{$row->city_id} {$row->city_dbname} status: {$status}, time: {$time}", true, true);
+	Wikia::log( __METHOD__, "info", $output, true, true);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)